packages/frontend/core/src/modules/doc/entities/property-list.ts

```diff
@@ -3,17 +3,34 @@ import {
   generateFractionalIndexingKeyBetween,
   LiveData,
 } from '@toeverything/infra';
+import { combineLatest, map } from 'rxjs';
 
 import type { DocCustomPropertyInfo } from '../../db/schema/schema';
+import type { FeatureFlagService } from '../../feature-flag';
 import type { DocPropertiesStore } from '../stores/doc-properties';
 
 export class DocPropertyList extends Entity {
-  constructor(private readonly docPropertiesStore: DocPropertiesStore) {
+  constructor(
+    private readonly docPropertiesStore: DocPropertiesStore,
+    private readonly featureFlagService: FeatureFlagService
+  ) {
     super();
   }
 
   properties$ = LiveData.from(
-    this.docPropertiesStore.watchDocPropertyInfoList(),
+    combineLatest([
+      this.docPropertiesStore.watchDocPropertyInfoList(),
+      this.featureFlagService.flags.enable_template_doc.$,
```

Comment:
Don't put feature flag check here.

put it https://github.com/toeverything/AFFiNE/blob/canary/packages/frontend/core/src/components/doc-properties/table.tsx#L204